### PR TITLE
check if current branch is develop

### PR DESCRIPTION
**Pull request by goncer**

### CODE REVIEW

**Review by gleb-urvanov — APPROVED** [review]
LGTM however would wait for review from @omaksi before merging

**Comment by omaksi** [comment]
But one thing that we could do is to refactor this script and break it into smaller ones

**Review by omaksi — APPROVED** [review]

**Comment by goncer** [comment]
OK! 
Will merge it tomorrow and create a task to refactor it. 
Makes sense to split it. I think that the time has arrived :)

**Pull request merged**
